api/app/routers/v1/chats.py

```diff
@@ -38,15 +38,15 @@ def update_chat(
     chat_id: str,
     chat_data: ChatCreate,
 ):
-    chat = Chat.read(id_=chat_id)
+    chat = Chat.read(chat_id)
```

Comment:
still need to update these to traverse scoped `actor.organization.chats` - so this'll get moved to a controller